packages/app/src/cli/services/web-env.ts

```diff
@@ -0,0 +1,77 @@
+import {selectApp} from './app/select-app.js'
+import {AppInterface} from '../models/app/app.js'
+import {output, file} from '@shopify/cli-kit'
+import {patchEnvFile} from '@shopify/cli-kit/node/dot-env'
+import {diffLines} from 'diff'
+
+type Format = 'json' | 'text'
+interface WebEnvOptions {
+  update: boolean
+  envFile: string
+}
+
+export async function webEnv(app: AppInterface, {update, envFile}: WebEnvOptions): Promise<output.Message> {
+  if (update) {
+    return updateEnvFile(app, envFile)
+  } else {
+    return outputEnv(app, 'text')
+  }
+}
+
+export async function updateEnvFile(app: AppInterface, envFile: WebEnvOptions['envFile']): Promise<output.Message> {
+  const selectedApp = await selectApp()
+
+  const updatedValues = {
+    SHOPIFY_API_KEY: selectedApp.apiKey,
+    SHOPIFY_API_SECRET: selectedApp.apiSecretKeys[0].secret,
+    SCOPES: app.configuration.scopes,
+  }
+
+  if (await file.exists(envFile)) {
```

Comment:
I think all this file manipulation would need to be relative to the App's root folder (it can be passed in via `--path` or the command could be run from a sub-folder)